crates/aiken-lang/src/builtins.rs

```diff
@@ -80,16 +84,16 @@ pub fn prelude(id_gen: &IdGenerator) -> TypeInfo {
     // Bool
     prelude.types_constructors.insert(
         BOOL.to_string(),
-        vec!["True".to_string(), "False".to_string()],
+        vec!["False".to_string(), "True".to_string()],
```

Comment:
Note: inverted to match the _actual_ convention followed by code-gen (constr 0 -> false, constr 1 -> true)